packages/commercetools/helpers/src/index.ts

```diff
@@ -1 +1,29 @@
-console.log('Hello commercetools helpers')
\ No newline at end of file
+import { UiMediaGalleryItem } from '@vue-storefront/interfaces'
+import { ProductVariant, Image, Category } from './types/GraphQL'
+
+export const getProductName = (product: ProductVariant): string => (product as any)._name
+
+export const getProductSlug = (product: ProductVariant): string => (product as any)._slug
+
+export const getProductPrice = (product: ProductVariant): number => product.price.value.centAmount
+
+export const getProductGallery = (product: ProductVariant): UiMediaGalleryItem[] =>
+  product.images.map((image: Image) => ({
+    small: '',
```

Comment:
if we dont have alternative sizes shouldnt we just provide same size everywhere? If someone will use small ones on one platform and change the platform to CT they will be blank. If we fill all with same image they will still be visible.